scripts/README.md

```diff
@@ -0,0 +1,94 @@
+# How to Run the Scripts for Training and Testing
+Contributor: Juan Antoni Barragan & Haoying(Jack) Zhou
```

Comment:
Small typo on my name! Missing an o.

Also, I think we should move your readme file to the docs folder and rename it to `running_train_and_test_scripts.md` or something similar.